HHS/Head-Start-TTADP
# Fix error in goals reducer, typo, include all topics in filters

## Description of change
This PR fixes an error in the logic where goals and objectives were fetched for approved ARs that could cause a site error.

It also:
- fixes a typo on the user profile page
- ensures that both AR topics (on older reports) and objective topics (on newer reports) are used in our activity report filters.
- updates the resource class to large in CircleCI jobs

## How to test
Visit /activity-reports/view/7199 on a prod-like environment and confirm that everything appears as normal.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-0
*  https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1340


## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [n/a] UI review complete

### Production Deploy


### After merge/deploy
